includes/class-omise-capabilities.php

```diff
@@ -52,8 +52,15 @@ public function getInstallmentBackends( $currency = '', $amount = null ) {
 	 *
 	 * @return string
 	 */
-	public function getBackends() {
-		return $this->capabilities->getBackends();
+	public function getBackends( $currency = '') {
+
+		$params   = array();
```

Comment:
spacing seems a little weird

Reply:
trying to follow this one 
any suggestions 🙏 
```php
	/**
	 * Retrieves details of installment payment backends from capabilities.
	 *
	 * @return string
	 */
	public function getInstallmentBackends( $currency = '', $amount = null ) {

		$params   = array();
		$params[] = $this->capabilities->backendFilter['type']('installment');

		if ( $currency ) {
			$params[] = $this->capabilities->backendFilter['currency']( $currency );
		}
		if ( ! is_null( $amount ) ) {
			$params[] = $this->capabilities->backendFilter['chargeAmount']( $amount );
		}

		return $this->capabilities->getBackends( $params );
	}
```

Reply:
ohh my screen is small I didn't see the spacing properly. I now see the equals sign lines up sorry!